modules/runtime/examples/docker/cluster/docker-compose.yml

```diff
@@ -0,0 +1,56 @@
+# Example on how to use Postgres 15 with Bonita in a cluster mode
+services:
+  bonita-db:
+    image: bonitasoft/bonita-postgres:15.4
+    ports:
+      - "5432:5432"
+    environment:
+      POSTGRES_PASSWORD: my-secret-pw
+    restart: always
+    # Uncomment those lines to mount a volume with your dumps
+    #volumes:
+    #  - ~/my/test/dumps:/opt/bonita/dump
+
+  bonita:
```

Comment:
in the case of an sca, the service should have the app name

Reply:
I agree but this is a generic example